docs/pages/guides/permissions.md

```diff
@@ -26,30 +28,40 @@ You can find the full list of available properties in [Apple's InfoPlistKeyRefer
 
 ### Managed workflow
 
-On Android, permissions are little bit simpler than iOS. In the managed workflow, permissions are controlled via the `android.permissions` property in your [**app.json** file](../../workflow/configuration.md#android). In the bare workflow, they have to be defined in your **AndroidManifest.xml**.
-
-Some Expo and React Native modules include permissions by default. If you use `expo-location`, for example, both the `ACCESS_COARSE_LOCATION` and `ACCESS_FINE_LOCATION` are implied and added to your app's permissions automatically.
+On Android, permissions are a bit simpler than on iOS. In the managed workflow, permissions are controlled via the `android.permissions` property in the [**app.json**](/versions/latest/config/app/#android).
 
-To limit the permissions your managed workflow app requires, set the `android.permissions` property in your [**app.json** file](../../workflow/configuration.md#android) to list only the permissions you need, and Expo will also include the minimum permissions it requires to run. See the [`Permission types`](#permission-types) below to learn about which Android permissions are added. You can find a full list of all available permissions in the [Android Manifest.permissions reference](https://developer.android.com/reference/android/Manifest.permission).
+Some Expo and React Native modules include permissions by default. For example, if you use `expo-location`, the `ACCESS_COARSE_LOCATION`, `ACCESS_FINE_LOCATION` and `FOREGROUND_SERVICE` permissions will be added to your app automatically. [See the `android.permissions` documentation](/versions/latest/config/app.md#permissions) to learn about which permissions are always included. To limit the permissions your managed workflow app requires, see [Excluding Android Permissions](#excluding-android-permissions).
```

Comment:
I've read the internal linking docs, but it didn't make clear to me about using relative file paths. I thought the arrows were use to provide a use case that this will happen when one uses links.

I guess, I should improve that to be more layman wording for folks like me (taking a note about it).

Thanks! I'll revert the link.